packages/shared/src/modules/ai-assistant/libs/enums/ai-assistant-validation-message.enum.ts

```diff
@@ -0,0 +1,13 @@
+const AiAssistantValidationMessage = {
+	CATEGORIES_REQUIRED: "Categories are required",
+	CATEGORY_ID_REQUIRED: "Category ID is required",
+	CATEGORY_NAME_REQUIRED: "Category name is required",
+	DESCRIPTION_REQUIRED: "Description is required",
+	DUE_DATE_INVALID_FORMAT: "Due date has an invalid format",
+	LABEL_REQUIRED: "Label is required",
+	TEXT_REQUIRED: "text  required",
```

Comment:
```suggestion
	TEXT_REQUIRED: "Text is required",
```
Use more descriptive messages